src/components/editor/use-host-bridge.js

```diff
@@ -87,23 +97,6 @@ export function useHostBridge( post, editorRef ) {
 		switchEditorMode,
 		undo,
 	] );
-
-	const postTitleRef = useRef( post.title );
-	const postContentRef = useRef( post.content );
-
-	useEffect( () => {
-		return subscribe( () => {
-			const { title, content } = window.editor.getTitleAndContent();
-			if (
-				title !== postTitleRef.current ||
-				content !== postContentRef.current
-			) {
-				onEditorContentChanged();
-				postTitleRef.current = title;
-				postContentRef.current = content;
-			}
-		} );
-	}, [] );
```

Comment:
This generic subscription is replaced by the `AutosaveMonitor`, which provides a more robust implementation, including intervals and dirty checks.